docs/writing-tests/import-stories-in-tests/stories-in-end-to-end-tests.mdx

```diff
@@ -61,7 +61,7 @@ With Playwright, you can write a test to check if the inputs are filled and matc
 
 Once you execute Playwright, it opens a new browser window, loads Storybook's isolated iframe, asserts if the inputs contain the specified values, and displays the test results in the terminal.
 
-#### Learn about other UI tests
+## Learn about other UI tests
```

Comment:
@shilman, when you're able, can you adjust this heading to the following so that it aligns with the other testing documentation?
```suggestion
**Learn about other UI tests**
```